tests/unit/src/unit/issues/Issue7497.hx

```diff
@@ -0,0 +1,82 @@
+package unit.issues;
+
+import haxe.io.FPHelper;
+import Math.isNaN;
+
+class Issue7497 extends unit.Test {
+#if js
```

Comment:
Why is this here?

Reply:
I think FPHelper should work across all targets, and if it doesn't it should be fixed accordingly. Changing it for all targets but then testing it only on JS is not good, even if it's the target that cares about it the most.

Could you remove that `#if js` so we can see if something fails?